src/api/get.js

```diff
@@ -0,0 +1,13 @@
+import { COMPONENTS_IMPLEMENTATION_MAP, panic } from '@riotjs/util'
+
+/**
+ * Get a custom tag implementation by name
+ * @param   {string} name - component name
+ * @returns {Object} implementation - tag implementation
+ */
+export function get(name) {
```

Comment:
This method name is too generic. Probably `getTagImplementationByName` would make more sense here.